Aardvark/ARKEmailBugReporter.m

```diff
@@ -133,7 +134,16 @@ - (void)_ARK_appendRecursiveViewControllerMappingToMapTable:(NSMapTable<UIView *
 @end
 
 
-@interface ARKEmailBugReporter () <CAAnimationDelegate, MFMailComposeViewControllerDelegate, UIAlertViewDelegate>
+@interface ARKDefaultPromptPresenter : NSObject <ARKEmailBugReporterPromptingDelegate>
+
+@property (nonatomic) ARKEmailBugReportConfiguration *configuration;
```

Comment:
we technically don't need it, but would be great to add nullability syntax here and in the property declaration below.